language/types_values_variables.md

```diff
@@ -269,7 +274,7 @@ Represents the abstract notion of "value", its subtypes are `Numeric`, `String`
 
 ### Numeric
 
-Represents the abstract notion of "number", its subtypes are `Integer`, and `Float`.
+Represents the abstract notion of "number", its subtypes are `Integer`, `Float`, `Timespan` and `Timestamp`.
```

Comment:
I think this is problematic as stated elsewhere. We would need a type that is Variant[Integer, Float] since that is what Numeric is now. We could add a "NumericBasedScalar" or similar of which Numeric is a sub type, but I am not sure that adds any value compared to just making Timespan and Timestamp be Scalars since other numeric based scalars would probably have other kinds of constraints in the arithmetic contract.